Makefile

```diff
@@ -1,6 +1,6 @@
 HURL_FILES = $(shell find ./rpc-call-examples/ -name '*.hurl')
 
-STARKNET_NETWORK?=katana
+STARKNET_NETWORK?=madara
```

Comment:
default to madara, whereby even if the user doesnt define `make build-and-deploy-kakarot STARKNET_NETWORK=madara`, it will default to it.